src/components/PlayerItem.vue

```diff
@@ -68,14 +79,18 @@ const handlePlayerSaidGabo = (playerName: string) => {
 .counter__blank-space {
   width: var(--btn-width);
 }
+
+.counter-btn {
+  width: var(--btn-width);
+}
 .container {
   width: 320px;
   margin: 0 auto;
   display: flex;
   flex-direction: column;
 }
-.button {
-  opacity: 0.1;
+.gabo-btn {
+  opacity: 0.6;
```

Comment:
not really good naming. here you put `gabo-btn` that sounds generic to me. maybe it should be a modifier. smth like `isDisabled`